sqeleton/databases/redshift.py

```diff
@@ -98,3 +109,15 @@ def query_table_schema(self, path: DbPath) -> Dict[str, tuple]:
             return super().query_table_schema(path)
         except RuntimeError:
             return self.query_external_table_schema(path)
+
+    def _normalize_table_path(self, path: DbPath) -> DbPath:
```

Comment:
@erezsh what do you think about: instead of overriding these methods, having a 3 part method in the base class

Reply:
Yes, I think that's a good idea.

Reply:
I will leave this change as is, but I can work on an additional PR to add a base class method + update the 3 part dbs to use that.